build.sh

```diff
@@ -115,6 +115,13 @@ if [[ "$PYTHON_VERSION" == "3.5" || "$PYTHON_VERSION" == "3.6" || "$PYTHON_VERSI
   popd
 fi
 
+
+"$PYTHON_EXECUTABLE" -m pip install -q setproctitle \
+        --target="$ROOT_DIR/python/ray/setproctitle_files"
+
+"$PYTHON_EXECUTABLE" -m pip install -q psutil \
+        --target="$ROOT_DIR/python/ray/psutil_files"
+
```

Comment:
Let's install `psutil` and `setproctitle` into `$ROOT_DIR/python/ray/thirdparty`